store/sqlstore/post_store.go

```diff
@@ -178,6 +178,11 @@ func (s *SqlPostStore) SaveMultiple(posts []*model.Post) ([]*model.Post, *model.
 			if ok {
 				post.ReplyCount += int64(count)
 			}
+		} else {
+			count, ok := rootIds[post.RootId]
+			if ok {
+				post.ReplyCount += int64(count)
+			}
 		}
```

Comment:
Nit 0/5 I'd rewrite it as:
```suggestion
	for _, post := range posts {
	    id := post.RootId
		if len(post.RootId) == 0 {
     		id := post.Id
		}
		count, ok := rootIds[id]
		if ok {
			post.ReplyCount += int64(count)
		}
		
```